_docs/style/colors.md

```diff
@@ -3,22 +3,17 @@ title: Color Palette
 layout: docs
 category: Style
 usage: |
-  The Helix color palette consists of deep modern hues juxtaposed with simple
-  monocromatic environments.
+  The Helix color palette consists of deep modern hues juxtaposed with simple monocromatic environments. Our color palette has been selected to enhance usability while adhering to accessibility criteria. Guidelines for color usage are outlined for individual components.
 preview-image: style/preview-color.svg
 resource: true
 status: complete
 ---
 
 {% include toc.html %}
 
-The use of color establishes cohesion across Rackspace products. Our color
-palette has been selected to enhance usability while adhering to accessibility
-criteria. Guidelines for color usage are outlined for
-[typography](/style/typography/) and individual components. Avoid adding color
-to "spice things up" as this will distract the user from his or her workflow.
 
-# Primary Colors
+
+### Primary Colors
```

Comment:
Any reason to start with L3 vs L2?

Reply:
@nimbinatus -- yep!  The page already has an H1 and H2.